spec/global_spec.rb

```diff
@@ -51,10 +27,9 @@
     end
 
     context 'when load from file' do
-      before { described_class.config_directory = File.join(Dir.pwd, 'spec/files/rspec_config') }
+      let(:config_path) { File.join(Dir.pwd, 'spec/files/rspec_config') }
 
       describe '#rspec_config' do
-        subject { super().rspec_config }
```

Comment:
So this spec only passed because of config pollution between specs. When I added cleanup (see spec_helper), it broke, and I confirmed that it breaks in `master`, too.

So if this example should lead to an `rspec_config` config key, and not putting the file's contents at the top level  - let's review the file loading logic, it doesn't work that way.